toolkit/scripts/pkggen.mk

```diff
@@ -280,6 +280,15 @@ compress-srpms:
 	tar -cvp -f $(BUILD_DIR)/temp_srpms_tarball.tar.gz -C $(SRPMS_DIR)/.. $(notdir $(SRPMS_DIR))
 	mv $(BUILD_DIR)/temp_srpms_tarball.tar.gz $(srpms_archive)
 
+# Seed the cached RPMs folder files from the archive.
+hydrate-cached-rpms:
+	$(if $(CACHED_PACKAGES_ARCHIVE),,$(error Must set CACHED_PACKAGES_ARCHIVE=))
```

Comment:
should this include the "y"?  ex: Must set CACHED_PACKAGES_ARCHIVE=y

Reply:
In this case `CACHED_PACKAGES_ARCHIVE` is expected to be a path. This check is a copy-paste of the `hydrate-rpms` target, so I kept the same message for consistency. I can update it to clarify, though, you're right.